ci-operator/config/Apicurio/apicurio-registry-system-tests/Apicurio-apicurio-registry-system-tests-main__service-registry-ocp4.13-lp-interop.yaml

```
@@ -0,0 +1,40 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  test-image:
+    name: apicurio-ci-tools
+    namespace: ci
+    tag: interop
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: service-registry-2-3
+  cron: 0 6 * * 1
```

```suggestion
  cron: 0 6 * * 1
  remote_api: true
```

Please add `remote_api: true` here as this will allow for us to trigger this job outside of the cron if needed.